streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java

```diff
@@ -393,10 +392,6 @@ private void shouldAddMetricsOnAllLevels(final String builtInMetricsVersion) thr
             IN_MEMORY_LRUCACHE_TAG_KEY,
             builtInMetricsVersion
         );
-        checkRocksDBMetricsByTag(
-            "rocksdb-state-id",
-            RecordingLevel.valueOf(streamsConfiguration.getProperty(StreamsConfig.METRICS_RECORDING_LEVEL_CONFIG))
-        );
```

Comment:
I moved the integration test verifications for RocksDB metrics into a new integration test.